Add community hours details to READMEs (#4305)
k0sproject/k0s: README.md

[Contributing Guide]: https://docs.k0sproject.io/stable/contributors/overview/
[Code of Conduct]:https://docs.k0sproject.io/stable/contributors/CODE_OF_CONDUCT/

### Community hours

> We will be holding regularly scheduled community hours, where anyone in the community is welcome to drop in and ask us questions, talk about projects, and chat.

Reviewer: I wonder if we could even drop "in the community" :thinking:

```suggestion
We will be holding regular community hours. Everyone in the community is welcome to drop by and ask questions, talk about projects, and chat.
```

> We currently hold the office hours call monthly, on last Tuesday of the month.

Reviewer: ```suggestion
We currently have a monthly office hours call on the last Tuesday of the month.
```

> To see the call details in your local timezone, check out [https://dateful.com/eventlink/2735919704](https://dateful.com/eventlink/2735919704).

Reviewer: ```suggestion
To see the call details in your local timezone, check out the [event on Dateful](https://dateful.com/eventlink/2735919704).
```
